packages/core/primitives/event-dispatch/src/eventcontract.ts

```diff
@@ -169,7 +169,7 @@ export class EventContract implements UnrenamedEventContract {
 
   constructor(
       containerManager: EventContractContainerManager,
-      private readonly stopPropagation = EventContract.STOP_PROPAGATION,
+      private readonly stopPropagation: false = false,
```

Comment:
Should we drop `: false` here?
```suggestion
      private readonly stopPropagation = false,
```

Reply:
`false` is a stricter type than what TypeScript would infer, which is `boolean`. Specifically, in TS, `false` allows passing only `false`, while `boolean` would allow `true` as well. The intention here is to remove this parameter altogether, so we want the more restrictive `false`. This mostly just catches in case someone tries to add a new use case in the next few days while I remove the options.

Reply:
Got it, thanks Tom!